src/Rules/Resources/HowToFix.resx

```diff
@@ -142,68 +145,68 @@
     <comment>The description of a problem with an element currently under inspection. The term "Window" should not be translated (i.e. it should be treated as a loanword).</comment>
   </data>
   <data name="HelpTextNotEqualToName" xml:space="preserve">
-    <value>The HelpText property of an element must not be the same as the element's Name property.</value>
-    <comment>The description of a problem with an element currently under inspection. The terms "HelpText" and "Name" should not be translated (i.e. they should be treated as loanwords).</comment>
+    <value>Write meaningful text for the element's UI Automation HelpText property. This text is typically similar to, or taken from, a tooltip or placeholder, such as "type here to search".</value>
+    <comment>Brief guidance on how to fix an accessibility problem. The term "HelpText" should not be translated (i.e. it should be treated as a loanword).</comment>
   </data>
   <data name="IsContentElementFalseOptional" xml:space="preserve">
-    <value>The recommended value of the IsContentElement property for the given control type is false. Please consider if this is an element that should be reported to an assistive technology user as content.</value>
-    <comment>The description of a problem with an element currently under inspection. The term "IsContentElement" should not be translated (i.e. it should be treated as a loanword).</comment>
+    <value>If this element does not contain data that is presented to the user (such as items in a list box or a button in a modal dialog box), set the IsContentElement UI Automation property to false.</value>
```

Comment:
There is a bit of mismatch on how we write this sentences. For some of them, we are saying the action but for others we are saying _If_. Is there something different between one check and the other (for example an automated check vs a warning)?

Reply:
Looking at the rule, it might be simpler to just say something like "Set the IsContentElement UI Automation property for this element to false". The rule's condition should already filter out controls where the user doesn't need to change anything.